src/js/components/todo/todoCount.js

```diff
@@ -0,0 +1,54 @@
+import {
+  $,
+  $all,
+  $closet,
+  $closetAttr,
+  TODO_SELCTOR,
+} from "../../utils/dom.js";
+import {
+  checkClassName,
+  checkLocalName,
+  removeClass,
+  setClass,
+} from "../../utils/eventUtils.js";
+
+export default function TodoCount(app) {
```

Comment:
```suggestion
export default function TodoCount(todoApp) {
```
위처럼 todoApp으로 적으면 좀더 명확할 것 같아요